### DISCUSSION TITLE
Boot LTSP client error

### DISCUSSION THREAD

**Discussion by Sistem-Pack** [ltsp/ltsp#880] [Q&A] [answered]
Hi, everyone! My thin client no load.  What kind of the problem could this?
![WhatsApp Image 2023-11-24 at 16 05 48](https://github.com/ltsp/ltsp/assets/56019760/2364163f-6fb7-40bf-8974-a32ba4824b39)

**Comment by alkisg** [comment]
Hi, are you trying to use RDB instead of the default NFS?

Also, what's the output of this command?

    parted /srv/ltsp/images/x86_64.img print

**Reply by Sistem-Pack** [reply]
> are you trying to use RDB instead of the default NFS?

I dont know =( Where can I'm watch this?

> Also, what's the output of this command?

![WhatsApp Image 2023-11-24 at 16 37 22](https://github.com/ltsp/ltsp/assets/56019760/7be3b346-a368-40b6-a103-a63248321bf0)

**Comment by alkisg** [comment]
How did you create x86_64.img, which command did you use?
Also, which distribution/version/kernel version are you using?

    cat /etc/os-release
    uname -a

**Reply by Sistem-Pack** [reply]
> Как вы создавали x86_64.img, какую команду использовали?

```
if ! grep -q "dns=default" /etc/NetworkManager/NetworkManager.conf; then
  sed -i '1 a dns=default' /etc/NetworkManager/NetworkManager.conf
fi
if ! grep -q "rc-manager=resolvconf" /etc/NetworkManager/NetworkManager.conf; then
echo "rc-manager=resolvconf" >> /etc/NetworkManager/NetworkManager.conf
fi

systemctl disable systemd-resolved
systemctl stop systemd-resolved
#ls -lh /etc/resolv.conf
#unlink /etc/resolv.conf
#echo nameserver 8.8.8.8 | sudo tee /etc/resolv.conf
rm /etc/resolv.conf
systemctl restart NetworkManager

add-apt-repository ppa:ltsp/proposed -y
apt update -y
apt install --install-recommends ltsp ltsp-binaries dnsmasq nfs-kernel-server openssh-server squashfs-tools ethtool net-tools epoptes -y
gpasswd -a ${SUDO_USER:-$USER} epoptes
install -m 0660 -g sudo /usr/share/ltsp/common/ltsp/ltsp.conf /etc/ltsp/ltsp.conf
pass=$(echo -ne "*******" | base64);
sed -i '/\[clients\]$/a \\nFSTAB_HOME="server:/home/shared /home/app nfs defaults,nolock 0 0"\n[**:**:**:**:**:**]\nHOSTNAME=PC\nAUTOLOGIN=LU\nPASSWORDS_PC="LU/_hui_"' /etc/ltsp/ltsp.conf
sed -i "s/_hui_/$pass/gi" /etc/ltsp/ltsp.conf
sed -i '/\[server\]$/a \\nNFS_HOME=1\nKERNEL_PARAMETERS="quiet"\nNAT=1\nIMAGE_TO_RAM=1\nMENU_TIMEOUT=-1\n' /etc/ltsp/ltsp.conf
ltsp dnsmasq
ltsp image /
ltsp ipxe
ltsp nfs
ltsp initrd
apt purge --yes --auto-remove indicator-application mate-hud snapd
apt install --yes synaptic
apt autoremove
```

> Кроме того, какой дистрибутив/версию/версию ядра вы используете?
![WhatsApp Image 2023-11-24 at 16 50 51](https://github.com/ltsp/ltsp/assets/56019760/19c26231-df68-497e-b919-26afd5ac0aba)

**Comment by alkisg** [comment]
Ah, you're using IMAGE_TO_RAM=1.
This was added by some developer in the past but they're not offering support for it, so I'll probably remove it in the future...

Try to remove `IMAGE_TO_RAM=1` from /etc/ltsp/ltsp.conf, and then run `sudo ltsp initrd`.

**Reply by Sistem-Pack** [reply]
Not worked =(
![WhatsApp Image 2023-11-24 at 17 12 59](https://github.com/ltsp/ltsp/assets/56019760/028f29b8-99f0-411c-8a72-170d2929037f)

**Comment by alkisg** [comment, marked as answer]
OK, better come to Matrix for online help via screen sharing for this:
https://ltsp.org/guides/chat-room/
